src/simulation/dynamics/linearTranslationalBodies/linearTranslationBodiesNDOF/linearTranslationNDOFStateEffector.rst

```diff
@@ -0,0 +1,126 @@
+
+Executive Summary
+-----------------
+
+The N-DoF linear translation body class is an instantiation of the state effector abstract class with N degrees of freedom. The integrated test is validating the interaction between the linear translation body module and the rigid body hub that it is attached to. In this case, a 4-DoF linear translation body has an inertia tensor and is attached to the hub by four single-degree-of-freedom axes. Each spinning axis is fixed in the parent's body frame and the effector is rigid, which means that its center of mass location does not move in the F frame. An optional motor force can be applied on the spinning axis, and the user can also lock the axis through a command. Moreover, the user can input a displacement reference that the effector will track through a spring and damper.
```

Comment:
make math variables like `N` and `F` typeset as math variables

Reply:
Done